src/theme/theme.ts

```diff
@@ -1037,9 +1041,9 @@ export const themeCustomized: Omit<Theme, 'applyStyles'> & CssVarsTheme =
             dark: '#FFFFFF',
           },
           surface3: {
-            light: '#F6F0FF',
-            main: '#F6F0FF',
-            dark: '#F6F0FF',
+            light: '#E5E1EB',
```

Comment:
make sure to double check it as it can be widely used across the app

Reply:
Yes, I have checked and it's only used in the `NavbarButtons`, though now I am wondering why I have updated it in this PR - might have overlooked doing the change in the previous one